src/components/snackbar/snackbar.tsx

```diff
@@ -32,7 +32,7 @@ export class Snackbar {
      * The amount of time in milliseconds to show the snackbar. Defaults to `5000` (5 seconds).
      */
     @Prop()
-    public timeout: number;
+    public timeout?: number;
```

Comment:
MDC seems to be doing this in the background. Read commit the message for more info.